### PR TITLE
Upgrade frontend to bs4

### PR DESCRIPTION
**Pull request by Katrienvh**
## Type
- Enhancement

## Pull request description
Finalize the conversion to Bootstrap 4 for the Backend.

Note: This branch is based continue-boostrap-4-integration on and has to be merged after #3065

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -14,21 +14,11 @@ gulp.task('build:backend:assets:copy-css-vendors', function () {
     'node_modules/bootstrap-tagsinput/dist/bootstrap-tagsinput.css',
     'node_modules/bootstrap-tagsinput/dist/bootstrap-tagsinput-typeahead.css',
     'node_modules/bootstrap-datepicker/dist/css/bootstrap-datepicker3.standalone.min.css',
-    'node_modules/cropper/dist/cropper.css',
-    'node_modules/bootstrap-accessibility-plugin/plugins/css/bootstrap-accessibility.css',
-    'node_modules/select2/dist/css/select2.css',
```
Comment (carakas) [inline gulpfile.js:19]: why did you remove select2?

File: src/Frontend/Themes/Fork/Modules/Blog/Layout/Templates/Detail.html.twig [hunk 2]
```diff
@@ -6,53 +6,49 @@
...
         </time>
       </div>
-      <div class="col-xs-12 col-md-6">
-        <div class="pull-right">
-          {% if item.allow_comments %}
-            {% if not item.comments %}
-              <a href="{{ item.full_url }}#{{ 'act.Comment'|trans }}" itemprop="discussionUrl">
+      <div>
+        {% if item.allow_comments %}
+          {% if not item.comments %}
```
Comment (sarahmey) [inline src/Frontend/Themes/Fork/Modules/Blog/Layout/Templates/Detail.html.twig:19]: The content of the if and else are basically the same, can't you write this is a nicer way?

File: src/Frontend/Themes/Fork/Modules/Blog/Layout/Widgets/RecentArticlesFull.html.twig [hunk 3]
```diff
@@ -15,14 +15,14 @@
           </h3>
 
           <div class="row">
-            <div class="col-xs-12 col-md-8">
+            <div class="col-12 col-md-8">
```
Comment (sarahmey) [inline src/Frontend/Themes/Fork/Modules/Blog/Layout/Widgets/RecentArticlesFull.html.twig:18]: Get rid of al the col-12's

File: src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
-    <li{% if loop.last %} class="active"{% endif %} itemprop="itemListElement" itemscope
-      itemtype="https://schema.org/ListItem">
-      {% if breadcrumb.url and not loop.last %}
-        {# set the full url or the item link will not validate #}
-        <a href="{{ SITE_URL }}{{ breadcrumb.url }}" title="{{ breadcrumb.title }}" itemtype="http://schema.org/Thing"
-          itemprop="item">
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb" itemscope itemtype="https://schema.org/BreadcrumbList">
+    {% for breadcrumb in breadcrumb %}
+      <li class="breadcrumb-item{% if loop.last %} active{% endif %}"{% if loop.last %} aria-current="page"{% endif %} itemprop="itemListElement" itemscope
```
[the remark this answers is not in the document]
Comment (Katrienvh) [reply src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig:5]: Is not possible because the first one is between the class quotes and second one outside that

File: src/Frontend/Themes/Fork/Core/Layout/Sass/components/_navbar.scss [hunk 5]
```diff
@@ -1,28 +1,38 @@
-.navbar-default {
+.navbar-dark {
+  background: $blue-dark;
```
[the remark this answers is not in the document]
Comment (Katrienvh) [reply src/Frontend/Themes/Fork/Core/Layout/Sass/components/_navbar.scss:2]: no not the background color

File: gulpfile.js (hunk 1 above)
Comment (Katrienvh) [reply gulpfile.js:19]: I think it was by accident
